src/OrchardCore.Modules/OrchardCore.Title/Views/TitlePart.Edit.cshtml

```diff
@@ -1,9 +1,15 @@
-@using OrchardCore.Title.ViewModels;
 @model TitlePartViewModel
+@using OrchardCore.ContentManagement.Metadata.Models
```

Comment:
Why this using ?

Reply:
Seems I added for a reason at the beginning, time to delete :)